src/platform/monitoring/record-event.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Helper function for reporting events to Google Analytics. An alias for window.dataLayer.push.
+ * @module platform/monitoring/record-event
+ * @see https://developers.google.com/tag-manager/devguide
+ * @param {object} data - The event data that will be sent to GA.
+ */
+
+export default function recordEvent(data) {
+  window.dataLayer.push(data);
```

Comment:
I think I saw some code that was returning `window.dataLayer.push()`, so we may want to return here to be consistent. Though I'm not sure why anyone would need that.